content/ops/deploying-the-admin-ui.md

```diff
@@ -80,4 +64,4 @@ Or...
 
 #### Operation
 
-Access the Admin UI at the uri specified in the deployment manifest. By default this is `https://admin.YOUR-SYSTEM-DOMAIN` as generated by `meta.subdomain "." properties.system_domain` in the `templates/admin-ui-deployment.yml`.
+Access the Admin UI at the uri specified in the deployment manifest. By default this is `https://admin.YOUR-SYSTEM-DOMAIN` as generated by `meta.subdomain "." properties.system_domain` in the `templates/admin-ui-deployment.yml`.  If you did not change this setting, try `https://admin.bosh-lite.com`.
```

Comment:
Not a blocker for this PR, but how much of this file could/should be moved to https://github.com/cloudfoundry-community/admin-ui-boshrelease/blob/master/README.md?